### PR TITLE
Add release version config

### PR DESCRIPTION
**Pull request by 0xSuperMan**
Added support for downloading latest or specific version

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -68,15 +70,33 @@ fi
...
 DIST="$VERSION"
 
+printf "Do you want the latest release(y/n): "
+read answer
+if [ "$answer" != "${answer#[Yy]}" ] ;then 
+        echo ""; 
+else
+        printf "You can find all the releases here $RELEASE_URL"
+        echo ""
+        printf "Enter relase version (e.g, v0.8.7, 0.8.6): "
```
Comment (samuelvanderwaal) [inline scripts/install.sh:80]: Looks like the `v` suffix is required so include it in the `0.8.6` example.

**Comment by samuelvanderwaal** [comment]
This is great, thank you! One small fix and I'll merge it.

File: scripts/install.sh (hunk 1 above)
Comment (0xSuperMan) [reply scripts/install.sh:80]: Yeah, I kind of thought the user is gonna copy paste it from github, but prefixed **v** is better

**Comment by 0xSuperMan** [comment]
Glad it worked :v:

File: scripts/install.sh (hunk 1 above)
Comment (0xSuperMan) [reply scripts/install.sh]: Fixed

**Comment by 0xSuperMan** [comment]
I have update the commit with the prefixed **v**. Do you want me to resolve the conflct?

**Pull request merged**